.github/workflows/build.yml

```
@@ -104,6 +104,15 @@ jobs:
         rm -rf /tmp/.buildx-cache
         mv /tmp/.buildx-cache-new /tmp/.buildx-cache
 
+    - name: Format Go code
+      run: |
+          OUTPUT=$(gofmt -l ./connectivity-exporter)
+          if [ -n "$OUTPUT" ]; then
+            echo "Please format the following files using gofmt"
+            echo "${OUTPUT}"
```

```suggestion
            echo "${OUTPUT}"
            gofmt -d ./connectivity-exporter
```

Idea: show the diff in the logs of the github action.